learn.html

```diff
@@ -203,12 +203,12 @@
 
             <section id="Data-By-Timestamp" class="group">
                 <h2>Extract data by Timestamp</h2>
-                    <p>Data from the Net Zero house includes measurements from various instruments around the house in one minute increments. Each variable, also called channels in  NIST documentation, provides the readings from one instrument, and the instruments are grouped into subsystems like elec, Electrical, DHW.</p>
+                    <p>Data from the Net-Zero house includes measurements from various instruments around the house in one minute increments. Each variable, also called channels in  NIST documentation, provides the readings from one instrument, and the instruments are grouped into subsystems like elec, Electrical, DHW.</p>
```

Comment:
@maabrown Is Elec repeated here, or is this two separate subsystems? `grouped into subsystems like elec, Electrical, DHW`